src/data_readers/data_reader_jag_conduit.cpp

```diff
@@ -1360,7 +1358,7 @@ std::vector<data_reader_jag_conduit::scalar_t> data_reader_jag_conduit::get_scal
 
   for(const auto key: m_scalar_keys) {
     std::string conduit_field = m_output_scalar_prefix + key;
-    std::string conduit_obj = '/' + pad(std::to_string(sample_id), SAMPLE_ID_PAD, '0') + '/' + conduit_field;
+    std::string conduit_obj = '/' + pad(std::to_string(sample_id), LBANN_SAMPLE_ID_PAD, '0') + '/' + conduit_field;
```

Comment:
Can `LBANN_DATA_ID_STR` be used as the others?

Reply:
Yes -- I thought I'd fixed all of those. Will fix after lunch